### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by bindsi**
Added instruction how to use the dockerfile to create an own container image to be referenced in the helm deployment #8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,8 +106,12 @@ Note this solution also allows for setting certificate based authentication, in
         network = "azure-iot-edge"
         ```
     - **Ctrl + X** to exit the editor, and enter **Y** to save your changes.
-3. Run `helm install --generate-name . --set publicSshKey="$(cat ~/.ssh/id_rsa.pub)",aziotEdgeVmEnableExternalSsh=true,aziotEdgeVmDiskSize=4Gi,nameOverride=aziot-edge-kubevirt,macAddress=fe:7e:48:a0:7d:22 --set-file azIotEdgeConfig=config.toml`
-4. Check and connect to the newly created VM by using cmds generated by the Helm chart install cmd i.e. `kubectl get vmi aziot-edge-kubevirt-linux` and `ssh ubuntu@$(kubectl get service aziot-edge-kubevirt-vm-ssh-service --output jsonpath='{.status.loadBalancer.ingress[0].ip}')`
+4. To create your own VM container image, you can use the dockerfile prepared [here](/deployment/Dockerfile) by running\
+   *docker build [OPTIONS] PATH | URL | -*\
+   and store it in your container registry or Docker Hub account.\
+   Afterwards, you can reference this image in the HELM deployment file [aziot-edge-data-volume-container.yaml](/deployment/helm/templates/aziot-edge-data-volume-container.yaml)
```
Comment (suneetnangia) [inline README.md:112]: HELM -> Helm?

File: README.md [hunk 2]
```diff
@@ -106,8 +106,12 @@ Note this solution also allows for setting certificate based authentication, in
         network = "azure-iot-edge"
         ```
     - **Ctrl + X** to exit the editor, and enter **Y** to save your changes.
-3. Run `helm install --generate-name . --set publicSshKey="$(cat ~/.ssh/id_rsa.pub)",aziotEdgeVmEnableExternalSsh=true,aziotEdgeVmDiskSize=4Gi,nameOverride=aziot-edge-kubevirt,macAddress=fe:7e:48:a0:7d:22 --set-file azIotEdgeConfig=config.toml`
-4. Check and connect to the newly created VM by using cmds generated by the Helm chart install cmd i.e. `kubectl get vmi aziot-edge-kubevirt-linux` and `ssh ubuntu@$(kubectl get service aziot-edge-kubevirt-vm-ssh-service --output jsonpath='{.status.loadBalancer.ingress[0].ip}')`
+4. To create your own VM container image, you can use the dockerfile prepared [here](/deployment/Dockerfile) by running\
```
Comment (suneetnangia) [inline README.md:109]: I'd add "Optionally, to create your own....

File: README.md (hunk 1 above)
Comment (Bindsi) [reply README.md:112]: oh, you are right --> I misused it when I read the path that includes helm

**Comment by suneetnangia** [comment]
Resolving issue #8

**Pull request merged**